indigo_app/templates/indigo_api/document/_toolbar.html

```diff
@@ -23,6 +23,7 @@
     <div class="dropdown-menu">
       <a class="dropdown-item" href="#defined-terms-modal" data-toggle="modal">Defined terms...</a>
       <a class="dropdown-item" href="#references-modal" data-toggle="modal">References...</a>
+      <a class="dropdown-item" href="#italics-modal" data-toggle="modal">Italics terms...</a>
```

Comment:
`Italicise terms...`?

Reply:
I was matching to 'Defined terms' and 'References' above, i.e. nouns as opposed to verbs?

Reply:
going with `Italicised` per in-person discussion